docs/testnet/readme.md

```diff
@@ -0,0 +1,27 @@
+# Onomy Testnet
+
+At [Onomy Protocol](https://onomy.io/), we are excited to launch our first public testnet for the Onomy Network (ONET)
+blockchain.
+
+As a Cosmos Tendermint-based chain, the ONET works via a Proof of Stake consensus mechanism, where the ANOM and it's 10^18 NOM representation is bonded to
+validators either directly or through delegation. The ONET will power the Onomy Exchange (hybrid DEX) and Onomy
+Reserve (stablecoin issuance), whilst operating with 100x the efficiency of Ethereum and facilitating inter-chain asset
+trading of NOM, Onomy's stablecoins, and other crypto assets.
+
+This iteration is also focused on proving out the Gravity bridge as well as deployment methodology and other custom
+modules for the upcoming production Onomy incentivized testnet and mainnet.
+
+The documentation here will guide you through the steps needed to run full nodes, become a validator, obtain ANOM/NOM, and
+try out the Onomy Network.
+
+To join us as a or discuss the current testnet, kindly get in touch with us via [Discord](https://discord.gg/27r73SYAkQ)
+. More focused discussions are possible on our [official forum](https://forum.onomy.io/).
```

Comment:
join us as a ?? or discuss the current testnet.......

Reply:
removed